.github/workflows/unit-test-runs.yml

```diff
@@ -80,8 +80,8 @@ jobs:
 
       - name: 'Run standalone unit tests'
         run: |
-          ctest --test-dir build -R '(^unit_|test_assert)' --no-tests=error
-          ctest --test-dir build -R 'test_ci_asserts'
+          ctest --test-dir build -R '(^unit_|test_assert)' --no-tests=error --output-on-failure
```

Comment:
We do have some rapidcheck tests in `tiledb/common/algorithm/test/unit_parallel_merge.cc` where we probably don't want to to this.

Can you add a `[rapidcheck]` tag to those tests, and then add that as a filter here so we run them separately without printing output?